recipes-bsp/u-boot/u-boot-2019.10.inc

```diff
@@ -0,0 +1,8 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/u-boot-2019.10:"
+
+SRC_URI_append = " \
```

Comment:
We should probably make this `SRC_URI_append_rpi` just in case it affects other machines.

Reply:
yes, updated